tests/plugins/test_base.py

```diff
@@ -42,17 +42,6 @@ def setUp(self):
 			device_matcher_udev,plugin_instance_factory,None,\
 			profiles.variables.Variables())
 
-	def test_get_effective_options(self):
-		self.assertEqual(self._plugin._get_effective_options(\
```

Comment:
This change should happen in the commit where you remove `_get_effective_options`.